VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
 \end{verbatim}
 \normalsize
-For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants the corresponding best estimate.
+For precise variants, END is POS + length of REF allele - 1.
 
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
 \end{verbatim}
 \normalsize
-This key can be derived from the REF/ALT fields but is useful for filtering. The reserved values must be used for the types listed below:
+SVTYPE is required for all structural variants. The following reserved values should be used whenever possible; indented values are subtypes that should be used if evidence supports them (see examples in Section 5.3, Encoding Structural Variants):\vspace{0.3cm}\\
 \begin{itemize}
-  \item DEL: Deletion relative to the reference
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
+\item [\null] INS:  Insertion relative to the reference
+	\begin{itemize}
+	\item [\null] MEI:  Mobile element insertion relative to the reference
+		\begin{itemize}
+		\item [\null] ALU:  Alu mobile element insertion relative to the reference
+		\item [\null] LINE1:  LINE1 mobile element insertion relative to the reference
+		\item [\null] SVA:  SVA mobile element insertion relative to the reference
+		\item [\null] HERV:  HERV mobile element insertion relative to the reference
+		\end{itemize}
+	\item [\null] RET:  Retrotransposon insertion relative to the reference
+	\end{itemize}
+\item [\null] DEL:  Deletion relative to the reference
+	\begin{itemize}
+	\item [\null] -MEI:  Absence of a mobile element present in the reference
+		\begin{itemize}
+		\item [\null] -ALU:  Absence of a Alu mobile element present in the reference
+		\item [\null] -LINE1:  Absence of a LINE1 mobile element present in the reference
+		\item [\null] -SVA:  Absence of a SVA mobile element present in the reference
+		\item [\null] -HERV:  Absence of a HERV mobile element present in the reference
+		\end{itemize}
+	\item [\null] -RET:  Absence of a retrotransposon present in the reference
+	\end{itemize}
+\item [\null] TRA:  Translocation
+\item [\null] STR:  Short tandem repeat variation
+\item [\null] INV:  Inversion relative to the reference
+\item [\null] DUP:  Duplication
+	\begin{itemize}
+	\item [\null] TANDUP:  Tandem duplication (one of Direct, Inverted-upstream, or Inverted-downstream)
+	\item [\null] DISDUP:  Dispersed duplication
+	\end{itemize}
+\item [\null] DELINS:  Deletion-insertion (replacement of reference sequence with sequence of different content or length)
+\item [\null] OTHER:  Complex variant not covered elsewhere
```

Comment:
Can't these be written as multiple VCF records with the same EVENT?

My preference is to just include simple variants in this field and use BND notation with EVENT to describe complex events.

OTHER is a catch-all that is going to break any program trying to be spec-compliant and I don't think belongs in these specifications.